enarx-keep-sgx-shim/src/handler.rs

```diff
@@ -217,7 +217,10 @@ impl<'a> Handler<'a> {
             .unwrap_or_else(|| self.aex.gpr.rdi.raw());
 
         loop {
-            unsafe { self.proxy(request!(libc::SYS_exit => code)) };
+            match unsafe { self.proxy(request!(libc::SYS_exit => code)) } {
+                Ok(_) => (),
+                Err(_) => (),
+            }
```

Comment:
Instead of trying to consume the result, which will leave a later reader of this code wondering what it does, use the `#[allow(unused_must_use)]` attribute which makes it clear that we are ignoring the return value on purpose.

Reply:
Also, if you did want to consume a variable, using `drop(value)` would be much cleaner.